benchmarks/dbmon/lib/memory-stats.js

```diff
@@ -46,9 +46,6 @@ var MemoryStats = function (){
 
 	var perf = window.performance || {};
 	// polyfill usedJSHeapSize
-	if (!perf && !perf.memory){
-		perf.memory = { usedJSHeapSize : 0 };
-	}
 	if (perf && !perf.memory){
```

Comment:
`perf &&` seems also useless.

Reply:
it may have effect when runtime not support performance api

Reply:
@JounQin 
get it, will make some changes

Reply:
@JounQin 
done, thanks